goth/runner/container/build.py

```diff
@@ -58,6 +57,14 @@ class YagnaBuildEnvironment:
     release_tag: Optional[str]
     """Release tag substring used to filter the GitHub release to download."""
 
+    @property
+    def is_using_binaries(self) -> bool:
```

Comment:
"using binaries" as opposed to "using `.deb` packages"? `.deb` packages also have binaries in them so it may be a bit confusing.

Reply:
Yes, `is_using_deb` is the only thing that came to my mind as well when I was thinking about a better name.